### PR TITLE
add command to view details of an item

### LINKED ISSUE #37: As a user, I want to be able to view detailed information about an item

### LINKED ISSUE #43: Update help link
Update help's link to our UG instead of the old addressbook UG

### PR DESCRIPTION
**Pull request by justacasul**
also changed the UG link

### CODE REVIEW

File: src/main/java/seedu/address/commons/util/StringUtil.java [hunk 1]
```diff
@@ -38,6 +38,27 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
                 .anyMatch(preppedWord::equalsIgnoreCase);
     }
 
+    /**
+     * Returns true if the {@code sentence} contains the {@code word}.
+     *   Ignores case, but a full word match is required.
+     *   <br>examples:<pre>
+     *       containsWordIgnoreCase("ABc def", "abc") == true
```
Comment (aqcd) [inline src/main/java/seedu/address/commons/util/StringUtil.java]: should these be containsMultipleWordsIgnoreCase?

File: src/main/java/seedu/address/ui/InventoryListPanel.java [hunk 2]
```diff
@@ -29,6 +28,15 @@ public InventoryListPanel(ObservableList<Item> itemList) {
         super(FXML);
         itemListView.setItems(itemList);
         itemListView.setCellFactory(listView -> new ItemListViewCell());
+
```
Comment (aqcd) [inline src/main/java/seedu/address/ui/InventoryListPanel.java]: remove line

File: src/main/java/seedu/address/ui/InventoryListPanel.java [hunk 3]
```diff
@@ -48,4 +56,21 @@ protected void updateItem(Item item, boolean empty) {
         }
     }
 
+
```
Comment (aqcd) [inline src/main/java/seedu/address/ui/InventoryListPanel.java]: remove line

File: src/test/java/seedu/address/logic/commands/ViewDetailsCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.item.NameIsExactlyPredicate;
+
+/**
+ * Contains integration tests (interaction with the Model) for {@code ViewDetailsCommand}.
+ */
+public class ViewDetailsCommandTest {
+
```
Comment (aqcd) [inline src/test/java/seedu/address/logic/commands/ViewDetailsCommandTest.java:24]: is it possible to test results of command? eg. if no such item exists, or check that the item being shown is indeed the right item

File: src/main/java/seedu/address/logic/commands/ViewDetailsCommand.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+            + "Example: " + COMMAND_WORD + "Bob's bone";
+
+    private final NameIsExactlyPredicate predicate;
+
+    public ViewDetailsCommand(NameIsExactlyPredicate predicate) {
+        this.predicate = predicate;
+    }
+
+    @Override
+    public CommandResult execute(Model model) {
```
Comment (khenghun) [inline src/main/java/seedu/address/logic/commands/ViewDetailsCommand.java:29]: Perhaps show a message to tell the user if no such item is found?

File: src/main/java/seedu/address/commons/util/StringUtil.java [hunk 6]
```diff
@@ -38,6 +38,27 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
...
+     */
+    public static boolean containsMultipleWordsIgnoreCase(String sentence, String words) {
+        requireNonNull(sentence);
+        requireNonNull(words);
+
+        String preppedWord = words.trim();
+        checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
+
+        return sentence.equalsIgnoreCase(preppedWord);
+    }
```
Comment (Rahul0506) [inline src/main/java/seedu/address/commons/util/StringUtil.java:60]: Would be better to name it as "exactlyMultipleWordsIgnoreCase"? "contains" is a little misleading.

**Review by Rahul0506 — COMMENTED** [review]
Small change in method name, other than that LGTM

**Pull request merged**